lib/fika/parser.ex

```diff
@@ -121,6 +121,14 @@ defmodule Fika.Parser do
     |> ignore(string("]"))
     |> Helper.to_ast(:exp_list)
 
+  tuple =
+    ignore(string("{"))
+    |> concat(allow_space)
+    |> concat(list_content)
```

Comment:
@emilsoman should we change `list_content` since its the same for tuples as well? 
Maybe call it as `enum_content` ?

Reply:
Tuple has a list-like content, so let's keep it as it is